package.json

```diff
@@ -17,8 +17,14 @@
   "devDependencies": {
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-flowtype": "^4.7.0",
```

Comment:
@chrismaree do you think this is a reasonable place to put these dependency imports? I couldn't figure out whether to add into `voter-dapp`'s package.json or here, but I saw there was already `eslint-config-prettier` here so I added.

Reply:
ye, I think this is fine for now. When we move to a more robust mono-repo structure this will be pulled into a package rather than at the base level and we can deal with it at the per package level.